src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerActivityPublisher.cs

```diff
@@ -30,9 +30,10 @@ public class LinuxContainerActivityPublisher : IHostedService, IDisposable, ILin
         private readonly HashSet<ContainerFunctionExecutionActivity> _uniqueActivities;
         private IDisposable _standbyOptionsOnChangeSubscription;
         private DateTime _lastHeartBeatTime = DateTime.MinValue;
-        private Timer _timer;
         private int _flushInProgress;
         private bool _initialPublish;
+        private CancellationTokenSource _publishingCts;
+        private Task _publishingTask;
```

Comment:
Can this be `readonly`? I don't see the one place (in the ctor) it is set.

Reply:
Based on the previous comment, I assume you're talking about the `_publishingTask`. If so, that is set when start is called, so we can't make this `readonly`.